README.md

```diff
@@ -87,6 +87,45 @@ docker run --gpus all \
 1. Copy the [compose.yml](./docker/compose.yaml) to your local machine
 2. Execute the command `docker compose up -d` in your terminal.
 
+### Method 5: Run on Clouds with SkyPilot
```

Comment:
```suggestion
### Method 5: Run on Kubernetes or Cloud VMs with SkyPilot
```

Reply:
"Run on Cloud VMs" feels like that we need to run on existing cloud VMs. How about just `Run on Kubernetes or Cloud with SkyPilot`?

Reply:
Sg, plural? `Run on Kubernetes or Clouds with SkyPilot`